### DISCUSSION TITLE
The navigation data are not enabled, bevor I install new SW (K1555) and M.I.B. by my Audi A3 MHIG navigation device

### DISCUSSION THREAD

**Discussion by Mokeju** [Mr-MIBonk/M.I.B._More-Incredible-Bash#313] [General]
Hello folks,
I have the following problem with my Audi A3 navigation device, it shows 'The navigation data are not enabled'. I then installed the software MHIG_EU_AU_K1555 on my Mib1. The installation took place without errors. Then I have the M.I.B. (Pre-V3.7.0 - MHIG Edition) installed and activated the patch. Patch is activatet. But unfortunately it still doesn't work. I have to add that my navigation device was already activated when I bought the vehicle and has worked for a number of years. Attached is the log file. I hope you can help me. Thank you.
[MU0415-LOG.txt](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/12360289/MU0415-LOG.txt)

**Comment by Mokeju** [comment]
It's also strange that there are no more activation codes, it says 'No functon enabling codes available', so it probably doesn't help if I install maps?

What is the SWDL_CLEANER all about, can you use it without hesitation?

After further attempts I now have the message 'Read error when accessing the navigation data'. But I still can't get the data on it because of the gray 'start update', the same with the software, what can I do to activate it again.
I still don't have an activation code on it and when I look at the log data, I notice the following errors:

### !!! Error: STAGE2SIZE in DB and RCC flash do NOT match

### !!! Generation of AddedFecs.tmp failed!

Can they have anything to do with this?

Can really nobody help me?

[MU0415-LOG.txt](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/12410036/MU0415-LOG.txt)

**Reply by avo-NL** [reply]
I am having the same problem. If you find a solutionThe navigation data are not enabled'. I then installed the software MHIG_EU_AU_K1555

**Reply by avo-NL** [reply]
Hello, I solved the problem.
Download the pre-release V3.7.1 "MHIG Edition", format the SD card as FAT32 and put it in it ( _Swdlautorun ) and change it to Swdlautorun. If you do the installation correctly, it will be easy.
https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/wiki/M.I.B-MENU-STRUCTURE
good luck

**Reply by colaas** [reply]
I have a very similar problem
**Generation of AddedFecs.tmp failed**

@Mokeju were you able to fix this?

**Comment by Burspoke** [comment]
Hi, can somebody put the firmware here please Mhig_eu_au_k1555
